cli/waiter/display.py

```diff
@@ -65,7 +65,8 @@ def tabulate_token_services(services, token_name, token_etag=None, show_index=Fa
                                           ('Last request', format_last_request_time(s)),
                                           ('Current?',
                                            format_using_current_token(s, token_etag or s.get('etag', None),
-                                                                      token_name))]
+                                                                      token_name)),
+                                          ('Scaling State', s['scaling-state'])]
```

Comment:
the code should be defensive against the value missing and display blank or `Unknown`. As written the code will throw an error if the `scaling-state` entry is missing.